dataset/all_time.json

```diff
@@ -0,0 +1,87 @@
+[
+   {
+      "rank": 1,
+      "name": "jiangjianfeng.jjf@antgroup.com",
+      "score": 2920
+   },
+   {
+      "rank": 2,
+      "name": "tate.thl@antgroup.com",
+      "score": 1107
```

Comment:
It is quite a surprise to me that my score is as high as 1107. My primary job is code review. I did submit some code. But I don't think my score could be even higher than Yuke Peng. Are you sure that the `score-and-rank-contributors` tool is implemented correctly?